.env

```diff
@@ -1,10 +1,10 @@
-REACT_APP_CONTRACTS_RELAY_HUB=0x86c406E582A360A1BfE363e28988Ae96780bEEe3
-REACT_APP_CONTRACTS_DEPLOY_VERIFIER=0x379E17449269350D1AD846833Dc0438415fe43Ca
-REACT_APP_CONTRACTS_RELAY_VERIFIER=0x22D26470BC3Ff3F0F2af176dF8c39ec5faF48274
-REACT_APP_CONTRACTS_SMART_WALLET_FACTORY=0x0c05f0C33D50A4758E50BC9ECA995eA097Fd4889
-REACT_APP_CONTRACTS_SMART_WALLET=0xFAF0d39D89D41A6A1E6D939cD55126a46a1E75B1
-REACT_APP_CONTRACTS_RIF_TOKEN=0xe639cbf39e10EEF13114006c378B1Aae8b9bD23c
-REACT_APP_CONTRACTS_RELAY_WORKER=0x860f96609aa2bab2fa3bec1d2279ea790e0b51d1
+REACT_APP_CONTRACTS_RELAY_HUB=0xE0825f57Dd05Ef62FF731c27222A86E104CC4Cad
+REACT_APP_CONTRACTS_DEPLOY_VERIFIER=0x1eD614cd3443EFd9c70F04b6d777aed947A4b0c4
+REACT_APP_CONTRACTS_RELAY_VERIFIER=0x5159345aaB821172e795d56274D0f5FDFdC6aBD9
+REACT_APP_CONTRACTS_SMART_WALLET_FACTORY=0x03F23ae1917722d5A27a2Ea0Bcc98725a2a2a49a
+REACT_APP_CONTRACTS_SMART_WALLET=0x73ec81da0C72DD112e06c09A6ec03B5544d26F05
+REACT_APP_CONTRACTS_RIF_TOKEN=0x726ECC75d5D51356AA4d0a5B648790cC345985ED
+REACT_APP_CONTRACTS_RELAY_WORKER=0x69862e277eab81e6a7ea6a7b387c39b2a996e533
```

Comment:
These changes aren't required and can be reverted

Reply:
leave as-is then :+1: